### ISSUE TITLE
The results of multiple runs are different

### ISSUE THREAD

**Issue by Midreancgle** [oushujun/EDTA#246]
Hello!

The EDTA is a great job! 
I know Repeatmodeler has a random sampling process, so I added the option “-srand” to ensure that the results of the repeatmodeler part are consistent.But when I run with the same sequence file, the same commands(--anno 1 --sensitive 0), the results are different.Can you give me some advices about this phenomenon?What should I do to ensure that the results of each run are consistent and  the results are relatively best.

**Comment by LQHHHHH** [comment]
I also found this question. Even if i use same parameters and same sequence the results are different between the two runs.

**Comment by oushujun** [comment]
Hi @Midreancgle and @LQHHHHH ,

Sorry, I just saw your message. The EDTA does things differently than RepeatModeler. It does not sample the input, but instead analyzes and annotates all input sequences, and randomly picks a sequence to represent others similar to it. The library-making process is totally random (`./util/cleanup_nested.pl`), which represents the uncertainty we are facing when trying to select for representative sequences. Fortunately, the differences are not very big as far as we observed. 

There are some other discussions here: #243 

Shujun
